app/models/twitter_account.rb

```diff
@@ -0,0 +1,45 @@
+class TwitterAccount
+  attr_reader(
+    :consumer_key,
+    :consumer_secret,
+    :access_token,
+    :access_token_secret
+  )
+
+  def self.config
+    Rails.application.config_for("twitter_accounts")
+  end
+
+  def self.find(slug)
+    new(config[slug])
+  end
+
+  def initialize(
+    consumer_key:,
+    consumer_secret:,
+    access_token:,
+    access_token_secret:
+  )
+    @consumer_key = consumer_key
+    @consumer_secret = consumer_secret
+    @access_token = access_token
+    @access_token_secret = access_token_secret
+  end
+
+  def tweet(copy)
+    TweetJob.perform_later(
+      consumer_key: consumer_key,
+      consumer_secret: consumer_secret,
+      access_token: access_token,
+      access_token_secret: access_token_secret,
+      copy: copy
+    )
+  end
+
+  def ==(other)
```

Comment:
Would it be better to also pass the `slug` in and then `==` based on the slug?

Reply:
@iHiD Ah yes. Good idea :)

Reply:
Whoop! :)